deepdow/utils.py

```diff
@@ -266,13 +274,16 @@ def raw_to_Xy(raw_data, lookback=10, horizon=10, gap=0, freq='B', included_asset
 
     asset_names = sorted(list(set(asset_names) - set(to_exclude)))
 
+    # ? What does absolute stand for? Variable could have a more meaninful name.
```

Comment:
It is supposed to stress that in the next step (`prices_to_returns`) we will get relative quantities. However, yeh, we can totally change the name.

Reply:
Makes more sense now, thank you. Maybe something like: `convert_to_returns` or `take_first_difference` , not sure.

Reply:
Makes more sense now, thank you.